Common/gfx/bitmap.cpp

```diff
@@ -256,9 +256,9 @@ static void read_palette(int bytes, RGB *pal, Stream *in, bool is_os2)
     int i, j;
 
     for (i=j=0; (i+3 <= bytes && j < PAL_SIZE); j++) {
-        pal[j].b = in->ReadByte() / 4;
-        pal[j].g = in->ReadByte() / 4;
-        pal[j].r = in->ReadByte() / 4;
+        pal[j].b = static_cast<uint8_t>(in->ReadInt8()) >> 2;

```

Comment:
I don't think it's proper or worth to replace `/4` with `>>2` here. Compilers today are smart enough to replace division on 2, 4 etc with bit shift, while with division here will be easier to understand the code intention.

Reply:
I will adjust this to use the division!